ckan/lib/helpers.py

```diff
@@ -797,7 +797,7 @@ def flash_notice(message, allow_html=False):
 @core_helper
 def flash_error(message, allow_html=False):
     ''' Show a flash message of type error '''
-    flash(message, category='alert-error', allow_html=allow_html)
```

Comment:
IIUC like this should go upstream too, right?

Reply:
@wardi yes, https://getbootstrap.com/2.3.2/components.html#alerts had to dig a little, but looks like they changed `alert-error` to `alert-danger` between BS2 and BS3